src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -1,7 +1,201 @@
+       :render-fn               profile-card}]]))
+
+(defn login-section
+  []
+  (let [{:keys [name customization-color error processing]
+         :as   multiaccount} (rf/sub [:multiaccounts/login])
+        sign-in-enabled?     (rf/sub [:sign-in-enabled?])
+        profile-picture      (:uri (first (:images multiaccount)))]
+    [rn/view
+     {:style style/login-container}
+     [quo/button
+      {:size                32
+       :type                :blur-bg
+       :icon                true
+       :on-press            #(reset! show-profiles? true)
+       :override-theme      :dark
+       :width               32
+       :accessibility-label :show-profiles
+       :style               style/multi-profile-button}
+      :i/multi-profile]
+     [quo/profile-card
+      {:name                name
+       :customization-color (or customization-color :primary)
+       :profile-picture     (when profile-picture {:uri profile-picture})
+       :card-style          style/login-profile-card}]
+     [quo/input
+      {:type                :password
+       :blur?               true
+       :override-theme      :dark
+       :placeholder         (i18n/label :t/type-your-password)
+       :accessibility-label :password-input
+       :auto-focus          true
+       :error?              (when (not-empty error) error)
+       :label               (i18n/label :t/profile-password)
+       :secure-text-entry   true
+       :on-change-text      (fn [password]
+                              (rf/dispatch [:set-in [:multiaccounts/login :password]
+                                            (security/mask-data password)])
+                              (rf/dispatch [:set-in [:multiaccounts/login :error] ""]))
+       :on-submit-editing   (when sign-in-enabled? login-multiaccount)}]
+     (when (not-empty error)
+       [quo/info-message
+        {:type  :error
+         :size  :default
+         :icon  :i/info
+         :style style/info-message}
+        (i18n/label :t/ops-wrong-password)])
+     [rn/keyboard-avoiding-view
+      {:style    style/keyboard-avoiding-view
+       :behavior :height}
+      [rn/view {:flex 1}]
+      [quo/button
+       {:size                40
+        :type                :ghost
+        :before              :i/info
+        :accessibility-label :forget-password-button
+        :override-theme      :dark}
+       (i18n/label :t/forget-password)]
+      [quo/button
+       {:size                40
+        :type                :primary
+        :customization-color (or :primary customization-color)
+        :accessibility-label :login-button
+        :override-theme      :dark
+        :before              :i/unlocked
+        :disabled            (or (not sign-in-enabled?) processing)
+        :on-press            login-multiaccount
+        :style               (style/login-button)}
+       (i18n/label :t/log-in)]]]))
 
 (defn views
   []
-  [:<>
-   [background/view true]])
+  (reset! show-profiles? true)
```

Comment:
could you elaborate why we reset atom here ?

Reply:
(this should by false by default, will change it)
So on starting app we should land on login screen (instead of profiles).
And if user toggles to profiles section and create an account and logout it will land again on profiles section but he/she should land on login section.

Reply:
shouldn't this be in app-db and manage by events ?

Reply:
Yes, We can move this to app-db, but this is kind of temporary. Currently, we don't have animations ready but the design team wants to create something [like](https://youtu.be/znIOqQLbNFk?t=79), (please check 1.23 time).
So this atom is temporary.

Reply:
So at some point instead of using reagent atom, we are going to use a shared value and animate between two states (profiles & login section)